Makefile

```
@@ -3,26 +3,21 @@ VERSION=$(shell python3 -c "import complex_colormap; print(complex_colormap.__ve
+upload: tag
 	@if [ "$(shell git rev-parse --abbrev-ref HEAD)" != "master" ]; then exit 1; fi
-	rm -f dist/*
-	python3 setup.py bdist_wheel --universal
+	rm -rf dist
+	python3 -m build
 	gpg --detach-sign -a dist/*
 	twine upload dist/*
 
 publish: tag upload
 
 clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)